src/_data/side-nav.yml

```diff
@@ -80,14 +80,18 @@
           permalink: /language/extend
         - title: Mixins
           permalink: /language/mixins
-        - title: Class modifiers
-          permalink: /language/class-modifiers
         - title: Enums
           permalink: /language/enums
         - title: Extension methods
           permalink: /language/extension-methods
         - title: Callable objects
           permalink: /language/callable-objects
+    - title: Class modifiers
+      children: 
+        - title: Overview & usage
```

Comment:
Out of curiosity, what style rules do you have about when to use "&" versus "and"?